### ISSUE TITLE
Explain that "not applicable" is a pass

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#219]
@maryjom, you requested this issue. The idea from our 2023-09-07 WCAG2ICT Task Force call

**Comment by maryjom** [comment]
The WCAG2ICT section [Comments on Conformance](https://w3c.github.io/wcag2ict/#comments-on-conformance) bullet #2 states the following, which I think is sufficient to address this:
>In the WCAG 2.2 conformance model, a success criteria is satisfied if the item being evaluated does not fail it. If the success criterion is in relation to something that does not exist for the item being evaluated (e.g. a success criterion is about captioning audio and there is no audio) then the success criterion is automatically met. This approach is central to the way the success criteria in WCAG are structured and worded.

If we want different wording to use the term "not applicable", it could instead read:
>In the WCAG 2.2 conformance model, a success criteria is satisfied if the item being evaluated does not fail it. If the success criterion is in relation to something that does not exist for the item being evaluated (e.g. a success criterion is about captioning audio and there is no audio), what some would call "not applicable", then the success criterion is automatically met. This approach is central to the way the success criteria in WCAG are structured and worded.

**Comment by mitchellevan** [comment]
+1 for the proposed edit

**Comment by maryjom** [comment]
Added this to a survey due 25 Oct.

**Comment by bruce-usab** [comment]
WCAG now has definition for [satisfies](https://www.w3.org/TR/WCAG22/#dfn-satisfies) which can reference.  I think that came out of the earlier WCAG2ICT work.

**Comment by maryjom** [comment]
@bruce-usab Good segway into the other survey where I am updating the "satisfies a success criterion" as a term definition. When this is implemented, the edit proposed here can link to that definition.

**Comment by maryjom** [comment]
This simply reiterates what is already stated in WCAG. No need for AG review.
